lib/utils.dart

```diff
@@ -30,49 +30,40 @@ import 'src/api/page_object_list_interface.dart';
 ///    Future submit() => exists(submitButton) ? submitButton.click() : null;
 ///
 
-// TODO: Consider using rootElementOf here, since many of these
-// utils duplicate its functionality.
-
 /// A matcher that checks if a PageLoaderElement/PageObject exists.
 /// If used on a List annotated by a Pageloader annotation, checks to see
 /// if not empty.
 bool exists(item) {
-  if (item is PageLoaderElement) {
-    return item.exists;
-  }
   if (item is PageObjectList) {
     return item.isNotEmpty;
   }
   try {
-    return item.$root.exists;
-  } catch (_) {}
-  return false;
+    return rootElementOf(item).exists;
+  } catch (_) {
```

Comment:
I think you can get rid of the `(_)` here and below...

Reply:
Getting an "expected identifier" error :(